src/GraphQL.ApiTests/GraphQL.approved.txt

```diff
@@ -265,6 +265,12 @@ namespace GraphQL
         public static T ToObject<T>(this System.Collections.Generic.IDictionary<string, object> source)
             where T :  class { }
     }
+    public struct ObjectProperty
```

Comment:
We can think of a better name.

Reply:
Can we just use `KeyValuePair<string, object>` instead of defining our own struct?

Reply:
No, because then the converter will start returning objects where arrays were previously returned. I specifically defined my own structure so that its use would clearly indicate the need to convert to an object. I updated comments.